### PR TITLE
chatgpt-client: new port

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

New port

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
@herbygillot @reneeotten This is also good to go. No idea why macOS 15 decided to cancel itself, but the port builds fine.

File: www/chatgpt-client/Portfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        LumitoLuma chatgpt-client bd4b13db19b45c548bb8ef81a13feb42ff495680
```
Comment (reneeotten) [inline www/chatgpt-client/Portfile:6]: as usual, this should use a tag not a commit hash

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: www/chatgpt-client/Portfile (hunk 1 above)
Comment (barracuda156) [reply www/chatgpt-client/Portfile:6]: How would I do that if it does not exist?

**Comment by barracuda156** [comment]
@reneeotten Bump

**Pull request merged**